python/dripline/core/gogol.py

```diff
@@ -35,7 +36,8 @@ def __init__(self, exchange='alerts', keys=['#'], name=None, **kwargs):
         logger.debug('Gogol initializing')
         if name is None:
             name = __name__ + '-' + uuid.uuid1().hex[:12]
-        Service.__init__(self, exchange=exchange, keys=keys, name=name, **kwargs)
+        Spimescape.__init__(self, exchange=exchange, keys=keys, name=name, **kwargs)
+        self._bindings.append(["alerts", keys[0]])
```

Comment:
This line needs to be a bit more sophisticated so that it deals with the case where multiple keys are provided (and should probably cope if someone gives a string instead of a list).
somewhere near the top something like:
```if isinstance(keys, str):
   keys = [keys]
```
then to modify self._bindings:
`self._bindings += [["alerts", a_key] for a_key in keys]`